packages/libs/react-ui/src/components/Notification/Notification.stories.tsx

```diff
@@ -21,6 +21,12 @@ const meta: Meta<
     },
   },
   argTypes: {
+    variant: {
+      options: ['standard', 'outlined'],
```

Comment:
Can be `options: Object.keys(displayVariants) as (keyof typeof displayVariants)[],`